src/others/sysmon/lv_sysmon.c

```diff
@@ -93,7 +96,65 @@ void _lv_sysmon_builtin_deinit(void)
 #endif
 }
 
-lv_obj_t * lv_sysmon_create(lv_obj_t * parent)
+#if _USE_PERF_MONITOR
+
+void lv_sysmon_show_performance(void)
```

Comment:
`create` works here as this AI cannot be interpreted as widget create API.
```suggestion
void lv_sysmon_create_performance(void)
```